### PR TITLE
Add endpoints to modify the scheduler state

### PR DESCRIPTION
**Pull request by viniciuschiele**
* Add endpoints to modify the scheduler state
* Requested on #179

### CODE REVIEW

**Comment by mxdev88** [comment]
Looks good.  Could you also make the `wait` option available? being able to tell `wait=False` when shutting down the scheduler would be helpful in my use cases.

https://apscheduler.readthedocs.io/en/stable/userguide.html#shutting-down-the-scheduler

Thanks

**Comment by christopherpickering** [comment]
@mxdev88 I added in a option to use post json to force a shutown.

```py
client.post(self.scheduler.api_prefix + '/shutdown', json={'wait':'False'})
```

File: docs/rst/api.rst [hunk 1]
```diff
@@ -10,6 +10,11 @@ Flask-APScheduler comes with a build-in API. This can be enabled/disabled in you
 
 
 - /scheduler [GET] > returns basic information about the webapp
+- /scheduler/pause [POST] > pauses job processing in the scheduler
+- /scheduler/resume [POST] > resumes job processing in the scheduler
+- /scheduler/start [POST] > starts the scheduler
+- /scheduler/shutdown [POST] > shuts down the scheduler with `wait=True`
+- /scheduler/shutdown [POST] + `json={'wait':'False'}` post data > shuts down the scheduler with `wait=False`
```
Comment (viniciuschiele) [inline docs/rst/api.rst:17]: it should be now `json={'wait':False}`

File: tests/test_api.py [hunk 2]
```diff
@@ -27,12 +27,80 @@ def test_scheduler_info(self):
...
+        response = self.client.post(self.scheduler.api_prefix + '/start')
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(self.scheduler.state, STATE_RUNNING)
+
+    def test_start_scheduler_already_running(self):
+        response = self.client.post(self.scheduler.api_prefix + '/start')
+        self.assertEqual(response.status_code, 400)
+
+    def test_shutdown_scheduler(self):
+        self.client.post(self.scheduler.api_prefix + '/start')
```
Comment (viniciuschiele) [inline tests/test_api.py:69]: The scheduler is already started in the `setUp` method, no need to call `/start` here

**Comment by christopherpickering** [comment]
@viniciuschiele we can fix the codecov fail on another pr maybe?

**Comment by viniciuschiele** [comment]
Yes, I've created this issue #223

**Pull request merged**